TSC-Charter.md

```diff
@@ -50,43 +67,54 @@ to continue attending TSC meetings as an observer.
+* Setting and maintaining appropriate standards for community discourse via the
+  various mediums under TSC control
+* Maintaining the list of additional Collaborators
+* Setting and maintaining standards covering contributions of code,
+  documentation and other materials
 * Mediating technical conflicts between Collaborators or Planetary Software Organization
-projects.
+projects
+
+The TSC serves as Planetary Software’s primary technical liaison
+body with external open source projects, consortiums, and groups.
+
+Many of these responsibilities will be delegated by the TSC to appropriate
+bodies such as the Working Groups.
 
-The TSC will define the Planetary Software Organization's release vehicles and serve as
-Planetary Software’s primary technical liaison body with external open
-source projects, consortiums and groups.
 
 ## Section 6. Planetary Software Organization Operations.
 
-The TSC will establish and maintain a development process for the Planetary Software Organization's Projects. The development process will establish guidelines
-for how the developers and community will operate. It will, for example,
-establish appropriate timelines for TSC review (e.g. agenda items must be
-published at least a certain number of hours in advance of a TSC
-meeting).
-
-There will be multiple Projects under the Planetary Software Organization organized by
-modules or subsystems. The TSC is responsible for organizing the Project
-structure, including possibly the creation and alignment of sub-Projects.
-Each Project must be within such policies as may be set by the TSC,
-have a well-defined scope and must work within that scope. The
-development process will provide for Projects to follow the lifecycle
-process as described in the [Project Lifecycle][] document. The development
-process will include a process for the TSC to oversee and approve changes
-in the lifecycle of a Project, which will include consideration of the
-following criteria:
+The TSC will establish and maintain a development process for the
+Planetary Software Organization's Projects. The development process
+will establish guidelines for how the developers and community will
+operate. It will, for example, establish appropriate timelines for
+TSC review (e.g. agenda items must be published at least a certain
+number of hours in advance of a TSC meeting).
+
+There will be multiple Top-Level Projects (TLPs) under the Planetary
+Software Organization as well as Top-Level Working Groups (TLWGs).
+The TSC is responsible for organizing the structure of these TLPs
+and TLWGs.  Each must be within such policies as may be set by the
+TSC, have a well-defined scope and must work within that scope. The
+development process will provide for TLPs to follow the lifecycle
+process as described in the [Project Lifecycle](Project-Lifecycle.md)
+document. The development process will include a process for the
+TSC to oversee and approve changes in the lifecycle of a Project,
```

Comment:
doesn't this sound like a TSC could kill a project if it doesn't like what it sees? Do we want this to sound as harsh?

Reply:
The term 'kill a project' is broad.  The TSC can't really stop human beings from working on a project, but the TSC can say "you guys are not conforming to our policies, and therefore you are no longer a 'Planetary Software Top Level Project' and we will remove you from our webpage, please stop claiming that you are endorsed by the Planetary Software Organization."

That being said, the part of the document that you indicate doesn't sound harsh to me.  Although maybe you're referring to the bullet points immediately below these lines.  Maybe the listed 'criteria' are too granular?  I wouldn't disagree with that.  Maybe we just remove the bullet list, and end the paragraph by replacing the last comma with a period, and getting rid of the rest?

Reply:
i personally like the list of bullet points, but maybe we should discuss these requirements more in the next TSC telecon? B/c i can see how, e.g. predictability of releases is a requirement that could be seen as scaring off peeps? Maybe they don't want to sign up for a release cycle? 
Above I was more concerned that the phrase "controlling the lifecycle of a project" sounds like there's the power of ending it. Can we rephrase it maybe so that it points more to its status within the lifetime as TLP somehow...
maybe:
Change "The development process will include a process for the TSC ...." sentence to:
Advances and other changes within the foreseen lifecycle stages as a TLP are overseen and approved by the TSC.

I like something like that more b/c it simply refers to a project's status as a TLP and not its own.